packages/chart/src/Pie.ts

```diff
@@ -436,7 +438,10 @@ export interface Pie {
     labelHeight(_: number): this;
     seriesPercentageFormat(): string;
     seriesPercentageFormat(_: string): this;
+    showLabels(): boolean;
+    showLabels(_: boolean): this;
 }
+Pie.prototype.publish("showLabels", true, "boolean", "If true, wedge labels will display");
 Pie.prototype.publish("showSeriesValue", false, "boolean", "Append data series value next to label");
 Pie.prototype.publish("seriesValueFormat", ",.0f", "string", "Number format used for formatting series values", null, { disable: w => !w.showSeriesValue() });
 Pie.prototype.publish("showSeriesPercentage", false, "boolean", "Append data series percentage next to label");
```

Comment:
You should probably disable the PPs which become irrelevant when showLabels is false.